n3fit/src/n3fit/backends/keras_backend/base_layers.py

```diff
@@ -17,34 +17,44 @@
     The names of the layer and the activation function are the ones to be used in the n3fit runcard.
 """
 
-from tensorflow.keras.layers import Lambda, LSTM, Dropout, Concatenate
-from tensorflow.keras.layers import concatenate, Input # pylint: disable=unused-import
+from tensorflow import expand_dims, math, nn
+from tensorflow.keras.layers import (  # pylint: disable=unused-import
+    Dropout,
+    Input,
+    Lambda,
+    concatenate,
+)
 from tensorflow.keras.layers import Dense as KerasDense
-from tensorflow import expand_dims
+from tensorflow.keras.layers import LSTM, Concatenate  # pylint: disable=unused-import
```

Comment:
`from tensorflow.keras.layers import` is done twice

Reply:
I agree this is a bit of a mess, I was surprised isort didn't pick it up, but in fact it's worse, isort caused this ^^
I think it wants to keep them alphabetical, but also not edit the pylint comment. In fact only `Input` requires it.

Of course it could be fixed by just having that comment on all of them, or removing it and using it somewhere just for that purpose, but I don't think that's an improvement.

Also I was confused by the `concatenate` (lower case `c`) import, I don't think that's used at all. There's `op.concatenate`, `np.concatenate` (using `tf.concat`), and there's the `Concatenate` layer which is accessed through `base_layer_selector` with the string `"concatenate"`, but this import is used nowhere.
So I removed it, here and in `backends/__init__.py`.

Do you agree on both points?

Reply:
It might have been used in the past and forgotten there.